.github/workflows/deploy.yml

```diff
@@ -42,5 +42,7 @@ jobs:
         run: vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
       # Trigger Algolia crawler
       - name: Crawl
+        uses: algolia/algoliasearch-crawler-github-actions@v1
         run: |
           curl -H "Content-Type: application/json" -X POST --user ${{ secrets.ALGOLIA_USER_ID }}:${{ secrets.ALGOLIA_API_KEY }} "https://crawler.algolia.com/api/1/crawlers/${{ secrets.ALGOLIA_CRAWLER_ID }}/run"
```

Comment:
We can remove this. From the docs I understand it only activates the crawler, it doesn't actually trigger a crawl till the next schedule. The below command should actually trigger a crawl.

Reply:
Yeah, seems correct 🤔 
```suggestion
```